package.json

```diff
@@ -58,7 +58,7 @@
       }
     },
     "start": {
-      "command": "babel-node bin/server",
+      "command": "npm run lint && babel-node bin/server",
```

Comment:
I agree that we need to run linter and tests as soon as new code is added if it doesn't slow down the process drastically.